staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -64,6 +68,16 @@ func (s *ServerRunOptions) ApplyTo(c *server.Config) error {
 	c.RequestTimeout = s.RequestTimeout
 	c.MinRequestTimeout = s.MinRequestTimeout
 	c.PublicAddress = s.AdvertiseAddress
+	c.FeatureGate = s.FeatureGate
+
+	m := map[string]bool{}
```

Comment:
//TODO indicating that this setting of global status is temporary until we can properly plumb the remaining feature gates.